### PR TITLE
191 Add feature request template

### PR DESCRIPTION
**Pull request by GeorgiChochov**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Background
+A clear and concise description of where the limitation lies.
+
+## Feature
+A description of the requested feature.
+
+### *Example:*
+A simple example if applicable.
+
+## Proposed Solution
```
Comment (benedeki) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I would mark this section as optional (maybe even the "Example" section).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
-**To Reproduce**
+## To Reproduce
 Steps to reproduce the behavior OR commands run:
 1. Go to '...'
 2. Click on '....'
 3. Enter value '...'
 4. See error
 
-**Expected behavior**
+## Expected behavior
```
Comment (Zejnilovic) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can we change this to behaviour? My spellcheck is going nuts today. 😄

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+---
+
+## Background
+A clear and concise description of where the limitation lies.
+
+## Feature
+A description of the requested feature.
+
+### *Example:* [Optional]
```
Comment (Zejnilovic) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I am ok with optional, but I think it is not needed. What is weird me though is that example is surrounded by ** and the proposed solution isn't and that it is on a different level. Why?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (GeorgiChochov) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fair enough

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (GeorgiChochov) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Good point, I wanted to place the `Example` as a subsection of the `Feature` section, but it just looks a bit odd, I guess.

**Pull request merged**